### ISSUE TITLE
NoneType object has no attribute 'replace' in opt_binary_expression_replace

### ISSUE THREAD

**Issue by joeavanzato** [thewhiteninja/deobshell#4]
Hi again,

Encountered another issue with a specific sample.

[git_134.ps1.txt](https://github.com/thewhiteninja/deobshell/files/10772069/git_134.ps1.txt

See Trace Below:
```
[02-17-2023 17:32:44] [DEBUG] Replace pipeline with single elements by CommandAst
[02-17-2023 17:32:44] [DEBUG] Remove unassigned variable use 'False'
[02-17-2023 17:32:44] [DEBUG] Remove unassigned variable use 'False'
[02-17-2023 17:32:44] [DEBUG] Remove unassigned variable use 'ServiceName'
Traceback (most recent call last):
  File "deobshell\main.py", line 86, in <module>
    main()
  File "deobshell\main.py", line 75, in main
    deob(OPTIONS['input'])
  File "deobshell\main.py", line 52, in deob
    o.optimize(ast)
  File "deobshell\modules\optimize.py", line 78, in optimize
    while optimize_pass(ast):
  File "deobshell\modules\optimize.py", line 63, in optimize_pass
    if opt(ast):
  File "deobshell\modules\optimizations\binary_expressions.py", line 75, in opt_binary_expression_replace
    formatted = target.replace(argument_values[0], argument_values[1])
AttributeError: 'NoneType' object has no attribute 'replace'
```
Please let me know if there is any way I can help.

**Comment by thewhiteninja** [comment]
fixed in 95db884521128431360f2d350fd7d9db8807a73e

**Comment by thewhiteninja** [comment]
fixed in 56f645477749508166b26c531b460d7ca9eaa568
